app/views/users/registrations/edit.html.erb

```diff
@@ -1,43 +1,67 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<div class="card ml-4 mr-4 xl:w-1/4 xl:mr-auto xl:ml-auto bg-gray-50 shadow-xl mt-12">
+  <div class="card-body">
+    <h1 class="text-lg font-medium">Edit <%= resource_name.to_s.humanize %></h1>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "users/shared/error_messages", resource: resource %>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+      <div class="form-control w-full mt-4">
+        <%= f.label :email, class: "label-text text-black mb-2 font-semibold" %>
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input input-bordered w-full" + (resource.errors[:email].any? ? " input-error " : " border-black ") %>
+        <% if resource.errors[:email].any? %>
+          <div class="field-error label-text mt-2 text-red-700">
+            <%= resource.errors.full_message(:email, resource.errors[:email].first) %>
+          </div>
+        <% end %>
+      </div>
 
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
-  <% end %>
+      <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+        <div class="label-text text-black mt-2">Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
```

Comment:
Just wanted to check: I never saw this "Currently waiting confirmation for: <email>" behavior in my testing. Is that expected?

Reply:
Got it! That makes sense.